src/constants.ts

```diff
@@ -86,3 +86,5 @@ var resources: any = {
 };
 
 _.set(window, 'OPENSHIFT_CONSTANTS.CATALOG_HELP_RESOURCES', resources);
+
+_.set(window, 'OPENSHIFT_CONSTANTS.NEW_PROJECT_MESSAGE', "Contact your system administrator at projectname@example.com to request a project.");
```

Comment:
That is what we do in the console, if no custom message is defined we fall back to a generic message defined in the source.  I would not put it in constants mainly because we don't want to give people two ways to customize that message and constants are supposed to be used for things people may want to override.